demo/src/routes/docs/[framework]/daisyUI/getMenu.ts

```diff
@@ -0,0 +1,28 @@
+import {daisyUIMetadata} from '$lib/components-metadata';
+import type {WidgetsConfig} from '@agnos-ui/svelte-bootstrap/config';
+
+export function getMenu(component: string) {
+	const componentCapitalized = (component.slice(0, 1).toUpperCase() + component.slice(1)) as Capitalize<keyof WidgetsConfig>;
+	const componentNames = Object.keys(daisyUIMetadata);
+	const index = componentNames.indexOf(componentCapitalized);
+	const isFirst = index === 0;
+	const isLast = index === componentNames.length - 1;
+
+	return {
+		...daisyUIMetadata[componentCapitalized],
```

Comment:
For consistency with the other `getMenu` fonction, maybe it would be better to explicitly write the keys you take from `daisyUIMetadata[componentCapitalized]`.